gitlens/gitlens-features.md

```diff
@@ -324,15 +324,18 @@ The Focus View allows you to stream line your workflow by providing a summarized
 
 Currently, this view is supported for GitHub repositories. In order to see the Focus View, you will need to connect the [GitHub Integration](/gitlens/gitlens-features/#connecting-the-github-integration).
 
-<img src="/wp-content/uploads/gl-focus-view-3.png" class="img-bordered img-responsive center">
+<img src="/wp-content/uploads/gl-focus-view-4.png" class="img-bordered img-responsive center">
 
-The Focus View includes the following sections:
-- My Pull Requests: Shows all PRs opened by you, assigned to you, or awaiting your review
-- My Issues: Shows all issues created by you, assigned to you, or that mention you.
+Within the Focus View, you can show Pull Requests and/or Issues that are opened by you, assigned to you, that need your review, or that mention you. Filters can be set to show only issues, pull requests, or all. Additionally, you can filter by mine, opened by me, assigned to me, needs my review, or mentions me. 
```

Comment:
```suggestion
Within the Focus View, you can show Pull Requests and/or Issues that are opened by you, assigned to you, that need your review, or that mention you. Filters can be set to show only issues, pull requests, or all.
```

The last sentence feels like it's repeating what the first sentence is already stating.